### ISSUE TITLE
oomusic Odoo version build

### ISSUE THREAD

**Issue by yonacwy** [DocMarty84/oomusic#8]
I have tried to install on oomusic on Odoo 12 & 13 does not work. Is there a version of oomusic i can use in Odoo as app so i can use the other Apps in Odoo?

**Comment by DocMarty84** [comment]
Hello,

What did you try exactly? The v3 branch is compatible with Odoo v13.

**Comment by yonacwy** [comment]
yeah i tried that branch. but i took oomusic app and put it in odoo 13
version i running on server.

**Comment by DocMarty84** [comment]
Did you update the app list? What doesn't work exactly?

**Comment by yonacwy** [comment]
hi, i justed tested oomusic app in odoo 12 community. the app appears but
with install errors.
